hms-app/src/components/admin-tools/HackathonForm.tsx

```diff
@@ -30,28 +30,26 @@ const useStyles = createStyles((theme) => ({
   },
 }))
 
-function NewHackathon() {
+type IProps = { context: string; hackathonID: string | null }
+
+function HackathonForm(props: IProps) {
+  const { context, hackathonID } = props
   const { classes } = useStyles()
   const today = new Date()
   const [startDateValue, setStartDateValue] = useState<Date | null>(new Date())
   const [endDateValue, setEndDateValue] = useState<Date | null>(new Date())
   const [hackathonText, setHackathonText] = useState({
```

Comment:
`hackathon` would be a better name, or draftHackathon possibly since the intention is to edit or create a hackathon

Reply:
ah, but now that I see how you modified this... you could just change this to `title` and do `useState('')` or something instead of making this an object